packages/fx-core/resource/package.nls.json

```diff
@@ -1050,8 +1050,8 @@
   "depChecker.needInstallFuncCoreTool": "You must have @NameVersion installed to debug your local functions.",
   "depChecker.failToInstallFuncCoreTool": "Failed to install @NameVersion. Install @NameVersion manually.",
   "depChecker.failToValidateFuncCoreTool": "Failed to validate @NameVersion after installation.",
-  "depChecker.globalFuncNodeNotMatched": "Your own Node.js(@NodeVersion) is incompatible with your own Azure Functions Core Tools(@FuncVersion). Please make sure your own Azure Functions Core Tools works with your Node.js version. See (@link) to Azure Functions supported Node.js versions.",
-  "depChecker.portableFuncNodeNotMatched": "Your own Node.js(@NodeVersion) is incompatible with Teams Toolkit Azure Functions Core Tools(@FuncVersion). Go to @Link to install Node.js (@RecommendedVersion is recommended).",
+  "depChecker.globalFuncNodeNotMatched": "Your own Node.js (@NodeVersion) is incompatible with your own Azure Functions Core Tools (@FuncVersion). Please make sure your own Azure Functions Core Tools works with your Node.js version. See (@link) to Azure Functions supported Node.js versions.",
```

Comment:
remove all "own" from sentences and the sentence next.
last sentence:     to ->  for

Reply:
Updated